server/src/test/java/com/google/coffeehouse/servlets/CreatePersonServletTest.java

```diff
@@ -0,0 +1,119 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.coffeehouse.servlets;
+
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.coffeehouse.util.IdentifierGenerator;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+/**
+ * Unit tests for {@link CreatePersonServlet}.
+ */
+public class CreatePersonServletTest {
+  private static final String NICKNAME = "Tim";
+  private static final String EMAIL = "test@fake.fake";
+  private static final String PRONOUNS = "he/him";
+  private static final String IDENTIFICATION_STRING = "predetermined-identification-string";
+
+  private CreatePersonServlet CreatePersonServlet;
+  private final LocalServiceTestHelper helper = new LocalServiceTestHelper();
+  private StringWriter stringWriter = new StringWriter();
+
+  @Mock private HttpServletRequest request;
+  @Mock private HttpServletResponse response;
+  
+  @Before
+  public void beforeTest() throws IOException {
+    helper.setUp();
+
+    IdentifierGenerator idGen = Mockito.mock(IdentifierGenerator.class);
+    Mockito.when(idGen.generateId()).thenReturn(IDENTIFICATION_STRING);
+    CreatePersonServlet = new CreatePersonServlet(idGen);
+
+    request = Mockito.mock(HttpServletRequest.class);
+    response = Mockito.mock(HttpServletResponse.class);
+    Mockito.when(response.getWriter()).thenReturn(new PrintWriter(stringWriter));
+  }
+
+  @After
+  public void afterTest() {
```

Comment:
Similarly to my other comment, could we change this function to tearDown()? We could also discuss this further as a group, but I currently have my tests written using the setUp and tearDown convention.